Resources/Prototypes/_Mini/tags.yml

```
@@ -2,4 +2,7 @@
   id: MagazineMiniMachineGunBox
 
 - type: Tag
-  id: CartridgeMinigun
\ No newline at end of file
+  id: CartridgeMinigun
+
+- type: Tag
+  id: ADT40mmGrenade
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)